### ISSUE TITLE
FR: amp-launch-bot

### ISSUE THREAD

**Issue by wassgha** [ampproject/amphtml#25551]
Currently a lot of the work involved in launching a feature is done manually 

A bot coupled with a launch tracking dashboard might come in very handy.

Ideally the bot would handle the following tasks:
- Claiming an experiment name and sending a PR to add the experiment to `experiments.js`
- Keeping track of a checklist of launch blocking items
- Tracking whether validation changes have made it to the production release
- Creating PRs to ramp up the experiment to avoid manually making PRs such as #24097 through a dashboard
- Create PRs to clean up experiments (and their usages)

**Comment by rcebulko** [comment]
The ramping-up bit is probably the lowest-hanging fruit; would be pretty easy to implement. The same could be said for claiming an experiment name.

A few questions:
1. What would the launch blocking checklist look like/is it the same for all launches?
2. How often are new features launched?
3. For tracking validation changes, are those usually submitted in a separate PR?

**Comment by wassgha** [comment]
1. It's different per-team but the [UI team's launch tracker](https://docs.google.com/spreadsheets/d/1RiiQrnQNSUS91svf3AAcqnd_xTuSOSUbHwreCEZUGJg/edit) gives you a pretty good idea on the tasks involved
2. `prod-config.json`'s [commit history](https://github.com/ampproject/amphtml/commits/master/build-system/global-configs/prod-config.json) (as well as `experiment.js`'s) can give us a good enough estimation. There's been around 30 commits to those files in the past 3 months
3.  Validator changes are usually submitted as part of the last implementation PR so that PR has to go into production before we can start allowing users to use it in the wild (otherwise their documents would be invalid AMP)
/cc @nainar & @sparhami who have more context than me on the launch process

**Comment by rsimha** [comment]
_[Infra triage]_ Is this something we'd like to do? If not, can this be closed?
